console/src/components/Services/RemoteSchema/Edit/Edit.js

```diff
@@ -242,6 +241,10 @@ class Edit extends React.Component {
           >
             <Common {...this.props} />
             {generateMigrateBtns()}
+            <small>
+              Note: Please make sure to rectify the errors present in the schema
+              before trying to modify it's details.
+            </small>
```

Comment:
@beerose added this note to sort of mitigate the problem that arises when we try to modify inconsistent schemas. I'll remove this if it is not required.

Reply:
I think it should only be present if remote schema is inconsistent. It may be confusing IMO.

Reply:
Updated this to be display msg on details page and disabling modify/delete btns